api/bin/chainflip-cli/src/main.rs

```diff
@@ -75,6 +75,23 @@ async fn run_cli() -> Result<()> {
 						.await?;
 					println!("Deposit Address: {address}");
 				},
+				Broker(BrokerSubcommands::WithdrawFees(params)) => {
+					let withdraw_details = api
+						.broker_api()
+						.withdraw_fees(
+							params.asset,
+							chainflip_api::clean_foreign_chain_address(
+								params.asset.into(),
+								&params.destination_address,
+							)?,
+						)
+						.await?;
+					println!(
+						"Withdrawal request successfull submitted. Tx hash: {:#x}",
+						withdraw_details.tx_hash
+					);
+					println!("Egress id: {:?}", withdraw_details.egress_id);
```

Comment:
```suggestion
					println!(
						"Withdrawal request successfully submitted: {}",
						withdraw_details
					);
```

(and implement Display for WithdrawalDetails.